### DISCUSSION TITLE
How do you overcome the out-of-date SQL Databases tutorial section?

### DISCUSSION THREAD

**Discussion by ddahan** [fastapi/fastapi#12075] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

Newcomer to FastAPI, I was following the official tutorial on the documentation.
However, at [this section](https://fastapi.tiangolo.com/tutorial/sql-databases/), the doc says it deals with Pydantic v1 and SQLAlchemy < 2. But I'm using the latest versions, and I want to learn how to do with SQLModel.

What would you suggest to keep learning at the right pace? I'm not sure I should read the whole documentation of SQLModel right now. Is there a beta version of this upcoming new documentation somewhere?
Thanks.

### Operating System

Linux

### FastAPI Version

0.112.1

### Pydantic Version

2.8.2

### Python Version

3.12.5

**Comment by RyanLilla** [comment]
is "bumping" still a thing? bumping this because I have also been following the tutorial, and I have also been wanting to know about the SQLModel-updated documentation. 🙏🏻

**Comment by sinisaos** [comment]
@ddahan I haven't used SQLModel (basically just a wrapper around SQLAlchemy and any SQLAlchemy tutorial about database querying and database management will work). I don't know when the documentation on SQL databases will be updated, but you can search Github for example projects that use FastAPI and SQLModel. Here are some basic examples
[https://github.com/testdrivenio/fastapi-sqlmodel-alembic](https://github.com/testdrivenio/fastapi-sqlmodel-alembic)
[https://github.com/anthonycepeda/fastapi-sqlmodel](https://github.com/anthonycepeda/fastapi-sqlmodel)
as well as some more complex examples (using domain, service layer, migrations etc.)
[https://github.com/petrgazarov/FastAPI-app](https://github.com/petrgazarov/FastAPI-app)
Hope that helps.

**Comment by ddahan** [comment, marked as answer]
@sinisaos thanks for your answer.
I found that reading the whole SQLModel documentation to replace section was very valuable.
